core/src/main/java/org/axonframework/config/ModuleConfiguration.java

```diff
@@ -54,4 +54,14 @@ default int phase() {
      * @see Configuration#shutdown()
      */
     void shutdown();
+
+    /**
+     * Returns the actual module configuration instance. Usually, it is the instance itself. However, in case of module
+     * configuration wrappers, we would like to provide the wrapped module configuration as the instance.
+     *
+     * @return the actual module configuration instance
+     */
+    default ModuleConfiguration instance() {
```

Comment:
Please call this `unwrap()` as it more clearly expresses the intent of this method.

Reply:
Liking the `ModuleConfiguration unwrap()` suggestion here